composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name" : "activecollab/jobqueuecommands",
+  "description" : "Active Collab job queue commands",
+  "license" : "proprietary",
+  "require" : {
+    "activecollab/jobsqueue": "~0.1",
+    "activecollab/databaseconnection": "~1.1",
+    "activecollab/configfile" : "~1.0",
```

Comment:
It is use in ActiveCollab\JobQueue\Commands\Instances\Command to get APPLICATION_UNIQUE_KEY